locales/fr-FR.json

```diff
@@ -86,6 +88,7 @@
     "activate": "Activer",
     "complete": "Compléter",
     "compose_desc": "Écrire un nouveau message",
+    "n-people-in-the-past-n-days": "",
```

Comment:
This seems to be a mistake from the Hungarian translation, it should have been with `_` like the one below. Will you remove all the keys you added that have a `-`?

Reply:
Indeed, that's more like a translation with i18n-ally that seems to failed, my bad for that.
Removed.